lib/graphql/schema/member/has_fields.rb

```diff
@@ -44,6 +45,34 @@ def field_class(new_field_class = nil)
           end
         end
 
+        # Call `method_name` as a wrapper around each of `field_names`.
+        # If no `field_names` are passed, `method_name` will be called around _all_ fields.
+        # @param field_names [Array<Symbol>]
+        # @param method_name [Symbol]
+        # @return [void]
+        def around_resolve(*field_names, method_name)
```

Comment:
This method signature seems a bit weird. I think it would make more sense as either:
```ruby
def around_resolve(method_name, *field_names)
end

# Usage:
# around all fields
around_resolve :upcase_value
# around just the listed fields
around_resolve :upcase_value, :name, :description
```
I think even more intuitive would be to borrow from rails `around_action`:
```ruby
def around_resolve(method_name, only: nil, except: nil)
end

# Usage:
# around all fields
around_resolve :upcase_value
# around just the fields listed
around_resolve :upcase_value, only: [:name, :description]
around_resolve :upcase_value, only: :name, :description
# around all except the fields listed
around_resolve :upcase_value, except: [:name, :description]
around_resolve :upcase_value, except: :name, :description
```